src/__tests__/useSafeTimeout.tsx

```diff
@@ -0,0 +1,20 @@
+import { renderHook} from '@testing-library/react-hooks'
```

Comment:
Not sure why prettier didn't catch this:
```suggestion
import { renderHook } from '@testing-library/react-hooks'
```